systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/StatefulSetUtils.java

```diff
@@ -98,9 +103,11 @@ public static boolean ssHasRolled(String name, Map<String, String> snapshot) {
      * @return The snapshot of the StatefulSet after rolling update with Uid for every pod
      */
     public static Map<String, String> waitTillSsHasRolled(String name, Map<String, String> snapshot) {
+        resourceOperationTimeout = ResourceManager.getResourceOperationTimeout(Kafka.RESOURCE_KIND, ResourceOperation.RESOURCE_ROLLING_UPDATE, snapshot.size());
```

Comment:
Why you put there Kafka kind? It should be more generic because it's `StatefulSetUtils`, not `KafkaUtils`